rfcs/text/0000-rfc-process-updates.md

```diff
@@ -0,0 +1,149 @@
+early collaboration, draft RFCs should post a title and problem statement or motivation. They need not be full fledged
+RFCs with every section populated.
+
+**rfc-deprecated** : This indicates that the design is no longer implemented or executed in the DCP and has been usurped
+by a newer design or process. Note that the deprecated tag is not standalone; it may be used in conjunction with 
+"rfc-approval" tag for example.
+
+**rfc-paused** : This indicates that an RFC has been paused for further refinement. This roughly implies that an RFC has
+gone from being in a review status to a "pre-rfc" status and should not be reviewed further until designated so.
+
+**rfc-withdrawn** : This tag is applied when an RFC has not finished the review process (and therefore has not been 
+explicitly rejected) but has been voluntarily withdrawn.
+
+Like the existing RFC process, any change of state to the RFC must be announced on the DCP slack channel. Also note that
+tagging an RFC has no implications on whether the PR is open or closed. Generally, RFCs in a terminal state (i.e.
+"approved", "declined", or "withdrawn").
+
+### 2. Reduce the RFC minimum review time
+
+The current RFC process requires a minimum of 2 weeks in DCP Community Review followed by 1 week of DCP Oversight
+Review. The intention of the "Community" review is to receive feedback not from other DCP team members, but rather the 
+scientific community.
+
+In practice, most RFCs are reviewed as a free-for-all during the community review period and then is followed by one
+week of radio silence during the "Oversight" review period since almost all feedback will have been resolved during the
+community review period.
+
+We propose removing Oversight review altogether. A two week minimum community review period still exists but the 
+"community" refers to all members of the DCP working on any of the chartered organizations instead of the scientific
+community only. This largely reflects the behavior of the review process today.
+
+Extensions to the two week minimum will remain the same as dictated by the existing RFC process at the discretion of
+the author(s) of the RFC.
+
+### 3. Add an explicit process to make "trivial" edits to published RFCs
+
+The original RFC process does not explicitly mention that minor edits (i.e. typos, re-wordings, etc.) are free to be
+pushed whenever needed. 
+
+We propose explicitly stating that if an RFC warrants minor/trivial edits, such edits may occur at any time by the
+author(s) of the RFC without announcement.
+
+### 4. Elucidate RFCs' implementation status
+
+Today, there is no requirement that RFCs must be related to any existing ticket in Zenhub which is usually a proxy for
+determining critical work to be done in a quarter. This means that, at times, RFCs are published/approved with no clear
+roadmap to completion which also weakens the implication of the RFC.
```

Comment:
I would use a different word than _roadmap_ in this case to avoid confusion with product roadmaps.

Reply:
Switched to "timeline" to match the section in the RFC template.

Reply:
Closing this thread as I believe it has been addressed, but feel free to re-open if not!